api/gentokens.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Loads env vars
+if [ -f ../local.env ]; then
+  source ../local.env
+else
+  echo "Global local.env file not found. Please create it."
+  exit 1
+fi
+
+# Checks if JWT_LIFETIME and SECRET_KEY are set
+if [ -z "$JWT_LIFETIME" ] || [ -z "$JWT_SECRET" ]; then
+  echo "JWT_LIFETIME and JWT_SECRET must be set in the global .env file."
+  exit 1
+fi
+
+# Checks if Docker Daemon is running
+if ! docker stats --no-stream &> /dev/null; then
+  echo "Docker Daemon not running ❌"
+  exit 1
+fi
+
+# Transpile TS to JS for tokenGen.ts 
+# Had troubles with ts-node, so I just used tsc
+tsc ./token/tokenGen.ts
+
+# Generate tokens for 'scrapy', and 'reWriter' modules
+SCRAPY_TOKEN=$(node -e "require('./token/tokenGen.js').generateAppToken('scrapy', '$JWT_SECRET', '$JWT_LIFETIME').then(console.log)")
```

Comment:
can we just hard code a random token in the `env` file rather than generating it each time. We don't need json web tokens for api keys as there's no need to serverlessly revoke access serverlessly

Reply:
When you mentioned "token" at the start of my development, I thought you wanted to use `JWT` ones. Will it make a lot of difference in our case if we keep it?

Reply:
Means you can remove _a lot_ of code by getting us devs to hard code a token.